packages/react-components/react-spinner/src/stories/Spinner/SpinnerAppearance.stories.tsx

```diff
@@ -1,10 +1,16 @@
 import * as React from 'react';
-import { makeStyles, shorthands, Spinner } from '@fluentui/react-components';
+import { makeStyles, shorthands, Spinner, tokens } from '@fluentui/react-components';
 
 const useStyles = makeStyles({
   container: {
     '> div': { ...shorthands.padding('20px') },
   },
+
+  // Inverted Spinners are meant as overlays (e.g., over an image or similar)
+  // so give it a solid, dark background so it is visible in all themes.
+  invertedWrapper: {
+    backgroundColor: tokens.colorPaletteBerryForeground1,
```

Comment:
Probably shouldn't use a foreground token as the background color. Does `colorNeutralBackgroundInverted` work?
```suggestion
    backgroundColor: tokens.colorNeutralBackgroundInverted,
```

Reply:
This is the color we used in ["filled appearance" stories elsewhere](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-input/src/stories/Input/InputAppearance.stories.tsx#L17) so I grabbed it here for consistency. We used it in the filled appearance stories because it maintains sufficient contrast with the filled lighter and darker appearances across all supported themes.

In this case since the inverted Spinner track is always white `colorNeutralBackgroundInverted` won't work because that token is white for the web dark theme.

I agree that we shouldn't use a foreground color here but given the track in this particular component and our usage elsewhere I propose we keep it for now and create an issue to follow up with a better solution to replace this usage and the filled appearance usages.

What do you think?

Reply:
Talked offline. Will hardcode the value for now so we aren't demonstrating foreground tokens for background colors. Will follow up with design on a long term solution that also work for the filled appearances.